### PR TITLE
[TRAFODION-1765] trafci would throw an error when set statistics on

### PR DESCRIPTION
**Pull request by arvind-narain**
TRAFODION-1765 - trafci would throw an error when set statistics on 
TRAFODION-1757 - 1) DCS stop script still not handling zookeeper error properly
                              2) No indication in DCS logs for "No Available Servers" unless
                                  debug level is enabled.

### CODE REVIEW

**Comment by hegdean** [comment]
Changes looks good to me.

File: dcs/src/main/java/org/trafodion/dcs/master/listener/ConnectReply.java [hunk 1]
```diff
@@ -275,12 +275,10 @@ boolean buildConnectReply  (Header hdr, ConnectionContext cc, SocketAddress clie
         if (found == false || exceptionThrown == true){
             exception.exception_nr = ListenerConstants.DcsMasterNoSrvrHdl_exn; //no available servers
             replyException = true;
-            if(LOG.isDebugEnabled()){
-                if (found == false)
-                    LOG.info(clientSocketAddress + ": " + "No Available Servers");
-                else
-                    LOG.info(clientSocketAddress + ": " + "No Available Servers - exception thrown");
-            }
+            if (found == false)
```
Comment (DaveBirdsall) [inline dcs/src/main/java/org/trafodion/dcs/master/listener/ConnectReply.java]: It looks like both (found == false) and (exceptionThrown == true) could be true. If they are both true, you will log "No Available Servers", but won't record that an exception is thrown. Is this what you want?

**Comment by DaveBirdsall** [comment]
Looks good to me.

File: dcs/src/main/java/org/trafodion/dcs/master/listener/ConnectReply.java (hunk 1 above)
Comment (arvind-narain) [inline dcs/src/main/java/org/trafodion/dcs/master/listener/ConnectReply.java]: Thanks Dave. Existing logic. Can I address this in the next checkin ?

Comment (DaveBirdsall) [inline dcs/src/main/java/org/trafodion/dcs/master/listener/ConnectReply.java]: Sure; I was just asking the question not suggesting that it was right or wrong.

**Comment by arvind-narain** [comment]
Yes Selva - I did confirm. - thanks for looking into this.

**Pull request merged**